src/Controllers/ModuleSuiteLocator.php

```diff
@@ -186,7 +186,7 @@ protected function loadSuiteConfiguration($suite, Module $module)
         // Resolve variables
         $resolvedConfig = $this->container->getParameterBag()->resolveValue($suiteConfig);
         return [
-            'type' => null, // @todo figure out what this is for
```

Comment:
I will assume that this is due to the initial not entirely correct architecture of the `GenericSuiteGenerator` class that we are using.
See: https://github.com/Behat/Behat/blob/f177773b9423f71f108ad7376119aade92cc1084/src/Behat/Testwork/Suite/Generator/GenericSuiteGenerator.php#L40-L43